### ISSUE TITLE
[BUG] Annotations for methods on Service Resources excess parameters

### ISSUE THREAD

**Issue by sfleischman105** [vemel/mypy_boto3_builder#72]
**Describe the bug**
Service Resources imported from `boto3-stubs` expect additional parameters that differ from the boto3 documentation.

For example, using the `Instance` service resource from `mypy-boto3-ec2` expects additional parameters for it's `start()` method. 

**To Reproduce**
1. Install 'boto3-stubs[...]'
2.  Run mypy on the following code sample

```python
import boto3
from mypy_boto3_ec2.service_resource import Instance

def start_instance(instance: Instance) -> None
    instance.start()
```
3. Actual mypy output

```
start_instance.py:5: Missing positional argument "InstanceIds" in call to "start" of "Instance"  [call-arg]
```

**Expected mypy-output**

I expected no linting errors. According to the [boto3.ec2.Instance.start](https://boto3.amazonaws.com/v1/documentation/api/1.17.11/reference/services/ec2.html#EC2.Instance.start) docs, the expected parameters are:

```
instance.start(
    AdditionalInfo='string',
    DryRun=True|False
)
```

So I believe `boto3-stubs` may be generating excess `Self` parameters for service resource objects, and in the case of `Instance` it expects a `List[InstanceIds]` to be passsed in as well. Other resource methods such as `Instance.stop()` and `Instance.terminate()` also differ from the boto3 due to this additional parameter.

**Additional context**
This is was on Ubuntu 18.04, with the following versions of the packages:
```
boto3-stubs==1.17.11
boto3==1.17.11
mypy-boto3-ec2==1.17.11
```

**Comment by vemel** [comment]
I believe this happens because `boto3` overrides these `botocore` methods. I will check, thanks!

**Comment by vemel** [comment]
FIxed in the latest release. Please let me know if it works as expected.

**Comment by sfleischman105** [comment]
I can confirm this is fixed. Thank you!
